exchanges/graphcache/src/store/data.ts

```diff
@@ -135,14 +139,15 @@ export const clearDataState = () => {
   if (process.env.NODE_ENV !== 'test' && !data.defer) {
     data.defer = true;
     scheduleTask(() => {
-      initDataState(data, null);
+      initDataState(OperationType.Read, data, null);
```

Comment:
I think this should be a write since we are writing data to the external store and potentially deleting unconnected links with `gc`

Reply:
Updated, thanks! What about the other two calls to `initDataState` (in `noopDataState` and `hydrateData`)?